reference/commands/cache.rst

```diff
@@ -158,30 +158,58 @@ conan cache clean
 .. code-block:: text
 
     $ conan cache clean -h
-    usage: conan cache clean [-h] [-v [V]] [-s] [-b] [-d]
-                             [-p PACKAGE_QUERY]
-                             pattern
+    usage: conan cache clean [-h] [-v [V]] [-s] [-b] [-d] [-t] [-p PACKAGE_QUERY]
+                             [pattern]
 
     Remove non-critical folders from the cache, like source, build and/or download
     (.tgz store) ones.
 
     positional arguments:
       pattern               Selection pattern for references to clean
 
-    optional arguments:
+    options:
       -h, --help            show this help message and exit
       -v [V]                Level of detail of the output. Valid options from less
                             verbose to more verbose: -vquiet, -verror, -vwarning,
                             -vnotice, -vstatus, -v or -vverbose, -vv or -vdebug,
                             -vvv or -vtrace
-
       -s, --source          Clean source folders
       -b, --build           Clean build folders
       -d, --download        Clean download folders
+      -t, --temp            Clean temporary folders
       -p PACKAGE_QUERY, --package-query PACKAGE_QUERY
                             Remove only the packages matching a specific query,
                             e.g., os=Windows AND (arch=x86 OR compiler=gcc)
 
+This command will remove non-critical files that Conan generates in its execution.
+It will remove all temporary files, along with all the source, build and download folders
+for every matching reference passed in *pattern*, unless a specific flag is supplied.
+
+
+**Examples**:
+
+
+- Remove all non-critical files:
+
+  .. code-block:: text
+
+      $ conan cache clean *
```

Comment:
doesn't work for me:
```
ERROR: Define at least one argument among [--source, --build, --download]
```

Reply:
Are you on the `release/2.0` branch? Works fine for me